connectors/v2/sullen-ural.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+/* global Connector */
+
+Connector.playerSelector = $('#tracks-async').length ? '#tracks-async' : ($('.band-col-right').length ? '.band-col-right' : '.album-tracks');
+
+Connector.getArtistTrack = function () {
+	var text = $('.play:contains("ll")').attr('download');
+	if (text) {
+		var separator = this.findSeparator(text);
+
+		var artist = null;
+		var track = null;
+
+		if (separator !== null) {
+			artist = text.substr(0, separator.index);
+			track = text.substr(separator.index + separator.length);
+		}
+
+		return {artist: artist, track: track};
```

Comment:
I would change this to have a single point of return - Perhaps init the track/artist vars outside of the first If. What do you think?